web/routing.go

```diff
@@ -220,10 +220,12 @@ func CreateSubdomainProxy(router *echo.Echo, serveApps echo.HandlerFunc) (*echo.
 
 // setupRecover sets a recovering strategy of panics happening in handlers
 func setupRecover(router *echo.Echo) {
-	recoverMiddleware := middleware.RecoverWithConfig(middleware.RecoverConfig{
-		StackSize:         1 << 10, // 1 KB
-		DisableStackAll:   !config.IsDevRelease(),
-		DisablePrintStack: !config.IsDevRelease(),
-	})
-	router.Use(recoverMiddleware)
+	if !config.IsDevRelease() {
```

Comment:
Because I feel that in dev, a panic should crash the stack. That way we do not miss it and also it is very useful to have the whole stacktraces dump of the go program when developing. With this middleware, the traces are truncated (only 1 KB) and the json formatting is frustrating.